### PR TITLE
Update cohort name

### PR DESCRIPTION
**Pull request by abbycabs**
Because it's Cacti now! 🌵 🌵 🌵

### CODE REVIEW

**Comment by beastlibrarian** [comment]
I second this!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # GitHub Playground
 We're playing with GitHub!
 
-A demo for Cohorts of Chaos \m/
+A demo for Cohort Cacti!
```
Comment (acabunoc) [inline README.md]: 🌵

Comment (abbycabs) [reply README.md]: what do you mean? very unclear

**Comment by acabunoc** [comment]
Great update! 👍 thank you for your help

**Pull request merged**